server/cache.ts

```diff
@@ -0,0 +1,21 @@
+import LRUCache from "lru-cache";
+
+const options = {
+  // the number of most recently used items to keep.
+  max: 2048,
+
+  // update the "recently-used"-ness of items on cache.has()
+  updateRecencyOnHas: true,
```

Comment:
just set updateAgeOnGet: true as well

even we don't use it (but see comment below), it's a good idea to add it in case later code add the get method and we forget to set it here.

Reply:
If someone or script GET our api repeatedly during the cache TTL, it might be hard to fetch the newest data if we set `updateAgeOnGet: true`. 
I think it's would be better to set the data TTL without `updateAgeOnGet`. wdyt

Reply:
I see what you mean. So how about this logic

1. Check cache if not hit, call GitHub API
2. If hit, then check the last fetched time, if it's less than a specified duration (say 2 hour), then we use the cache, otherwise we call the API and update the star count in the cache?

Reply:
LGTM. And I think the [lru-cache option](https://github.com/isaacs/node-lru-cache#options) with `ttl: 2 hour, updateAgeOnGet: false` would works right like this. So we don't need to record the last fetch time in cache value.